kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/configuration/NetworkConfigurationServicePropertiesBuilder.java

```diff
@@ -104,6 +105,37 @@ private void setIpv4Properties() {
         }
     }
 
+    private void setIpv6Properties() {
+        this.properties.setIp6Status(this.ifname, this.gwtConfig.getIpv6Status());
+        this.properties.setIp6AddressMethod(this.ifname, this.gwtConfig.getIpv6ConfigMode());
+
+        boolean isManual = this.gwtConfig.getIpv6ConfigMode().equals("netIPv6MethodManual");
+        boolean isWan = this.gwtConfig.getIpv6Status().equals("netIPv6StatusEnabledWAN");
+        boolean isAuto = this.gwtConfig.getIpv6ConfigMode().equals("netIPv6MethodAuto");
+
+        if (isWan) {
+            if (!Objects.isNull(this.gwtConfig.getIpv6WanPriority())) {
```

Comment:
Really nitpicking here: for readability reasons I would avoid negating the call and directly use the `nonNull` method.

Don't feel obliged to change this. I would merge this as-is since it's a really small thing.

```suggestion
            if (Objects.nonNull(this.gwtConfig.getIpv6WanPriority())) {
```